src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -397,8 +402,69 @@ def makeSubscriptions(self, workflow):
             pass
 
         elif self.msConfig['defaultDataManSys'] == 'Rucio':
-            pass
+            if isinstance(workflow, MSOutputTemplate):
+                msg = "Making transfer subscriptions for %s"
```

Comment:
I find it more readable like this, and it is also following the same pattern as above (for the DDM data management system ): 
https://github.com/dmwm/WMCore/blob/b8326f9e07ef9cb59e126dafb0008f72c8767845/src/python/WMCore/MicroService/Unified/MSOutput.py#L323-L324

Reply:
Yes, and I use it too. However, I usually write it in multiple lines if we would end up with a line too long. For this case, the line is fairly short
```
self.logger.info("Making transfer subscriptions for %s", workflow['RequestName'])
```
and having all the print related statement in a single line looks better (in addition to counting to less LOC).

This would also be consistent with the way logging has been implemented in WMCore over the last few years. The more uniformity we can agree in the code, the better IMO.

Reply:
Please update it.

Reply:
Done

Reply:
Done